modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/reconstructedpatchedcell/v0.1.0.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
This schema should extends the neurosciencegraph/morphology/reconstructedcell/v0.1.0.json one